common/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -4,3 +4,5 @@ databaseChangeLog:
       file: db/changelog/v001/schema.sql
   - include:
       file: db/changelog/v001/seed_data.sql
+  - include:
+      file: db/changelog/v001/ab2d-599.sql
```

Comment:
Including a super short description of a change set within the file name would be good for traceability, i.e. `ab2d-599-rename-consent.sql`

Reply:
done